### PR TITLE
docs(Accounts): Process Statement Of Accounts

### PR DESCRIPTION
**Pull request by abhishekbalam**
Docs for https://github.com/frappe/erpnext/pull/22901

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/accounts/index.html [hunk 1]
```diff
@@ -107,6 +107,11 @@
 				'title': 'Accounting Reports',
 				'link': '/docs/user/manual/en/accounts/accounting-reports',
 				'content': 'ERPNext Accounting module has multiple built in reports like Profit and Loss, Balance Sheet, etc.'
+			},
+			{
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/index.html:111]: also add to contents.md

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<!-- add-breadcrumbs -->
+
+# Process Statement Of Accounts
+
+###Process Statement Of Accounts is a tool which helps you send Statement Of Account (General Ledger Report) and Ageing (Accounts Receivable Summary) Report as a PDF to your customers in bulk via email either manually or automated periodically.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:5]: bold not h3

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### 3.1 Option to download consolidated PDF of all customers
+
+On creating an entry, there is a button seen at the top called "Download" which lets you see the consolidated report PDF of all customers for reviewing.
+
+### 3.2 Option to send emails manually
+
+On creating an entry, there is a button seen at the top called "Send Emails" which lets you trigger email sending manually to the customers. The emails are queued via a background job, which you can track in the "Email Queue" doctype with the DocType and Document references.
+
+## 3.3 Using dynamic values in the Email Subject and Body
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:77]: h3

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/template.png">
+
+Resulting Email:
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/email.png">
+
+Report PDF:
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/report.png">
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:95]: add related topics at the end

**Review by prasadarr — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md (hunk 2 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:5]: Done

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+> Introduced in Version 13
+
+**Process Statement Of Accounts is a tool which helps you send Statement Of Account (General Ledger Report) and Ageing (Accounts Receivable Summary) Report as a PDF to your customers in bulk via email either manually or automated periodically.**
+
+<br>
+To access *Process Statement Of Accounts* list you can search "Process Statement of Accounts" in the navbar.
+
+## 1. Prerequisites
+
+1. The tool needs to use the email-ids of customers to send them the reports. On not finding the below email entries in the Customer contacts the tool won't allow you to select the respective customer, so please make sure the following details are filled in the customer documents.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:16]:
```suggestion
1. The tool needs to use the email IDs of customers to send them the reports. On not finding the below email entries in the Customer contacts the tool won't allow you to select the respective Customer, so please make sure the following details are filled in the customer documents.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+    - Billing Email of Customer: This is mandatory and can be set within the [customer contact](/docs/user/manual/en/CRM/contact#1-how-to-create-a-contact) with "Is Billing Contact" option checked.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:18]:
```suggestion
    - Billing Email of Customer: This is mandatory and can be set within the [Customer Contact](/docs/user/manual/en/CRM/contact#1-how-to-create-a-contact) with "Is Billing Contact" option checked.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    - Primary Email of Customer: This is not mandatory, unless you select the "Send To Primary Contact" in the form.
+
+2. Email Account setup with outgoing enabled. Learn more about this [here](/docs/user/manual/en/setting-up/email/email-account).
+
+
+## 2. How to create a Process Statement Account entry
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/name_and_filters.png">
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:26]: show image later not first

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+
+1. Go to the "Process Statement Of Account" list view by searching in the navbar and click on "New".
+
+2. Enter a name for the entry, for future reference.
+
+3. Set the General Ledger filters for the statements which will be sent to the customers.
+
+    - "From Date" and "To Date" filters will be hidden and auto-filled dynamically when "Enable Auto Email" option is selected.
+    - "Project" and "Cost Center" fields are multi-select.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:36]: what does multi select mean? You're not talking to your dev buddies

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    
+    - On selecting "Enable Auto Email" you will see the options to send automated periodic reports to the customers in the entry.
+
+        - You can select the "Frequency" at which the emails will be sent after the "Start Date" to the customers.
+        - You can also select the "Filter Duration", which is the gap between the "From Date" and "To Date" values in the General Ledger report until the date of sending the report ("To Date").
+        - These mails are not sent right away, but at midnight as a background process.
+    
+    - After this you can select the "Subject", "CC To" and "Body" fields of the email. If you don't set values to this field, default values will be used like shown in the help box below the body field.
+    
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/email-content.png">
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:65]: email content here is empty, show the same text shown in the inbox screenshot

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+7. Review your settings and click on "Save"
+
+## 3. Features
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/buttons.png">
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:71]: image later not before

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+### 3.2 Option to send emails manually
+
+On creating an entry, there is a button seen at the top called "Send Emails" which lets you trigger email sending manually to the customers. The emails are queued via a background job, which you can track in the "Email Queue" doctype with the DocType and Document references.
+
+### 3.3 Using dynamic values in the Email Subject and Body
+
+You can use Jinja tags to enter dynamic values from:
+
+- the customer to which the email will be sent to under the "customer" object
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:85]: caps when starting a sentence

**Review by prasadarr — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+    
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/email-content.png">
+
+7. Review your settings and click on "Save"
+
+## 3. Features
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/buttons.png">
+
+### 3.1 Option to download consolidated PDF of all customers
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:73]:
```suggestion
### 3.1 Download consolidated PDF of all customers
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+On creating an entry, there is a button seen at the top called "Download" which lets you see the consolidated report PDF of all customers for reviewing.
+
+### 3.2 Option to send emails manually
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:77]:
```suggestion
### 3.2 Send emails manually
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
+<!-- add-breadcrumbs -->
+
+# Process Statement Of Accounts
+
+> Introduced in Version 13
+
+**Process Statement Of Accounts is a tool which helps you send Statement Of Account (General Ledger Report) and Ageing (Accounts Receivable Summary) Report as a PDF to your customers in bulk via email either manually or automated periodically.**
+
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:8]: add more context here, why this feature would be used? where?

**Review by prasadarr — CHANGES_REQUESTED** [review]

**Comment by prasadarr** [comment]
remove subfolder from images "processed.....", keep images directly under accounts

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md (hunk 8 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:36]: https://docs.erpnext.com/docs/user/manual/en/customize-erpnext/articles/table-multiselect-field.md

I'll link this so people get familiar with the field type.

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md (hunk 10 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:71]: Added becuase the 1st two points needs its reference.
Will move it to after the first 2 points.

File: erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md (hunk 9 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md:65]: I'll add the correct one, but it wont be same as the inbox.
This pic shows the default values, the emails will show what we customised below.

**Comment by abhishekbalam** [comment]
> remove subfolder from images "processed.....", keep images directly under accounts

Removed.

**Review by prasadarr — APPROVED** [review]

**Pull request merged**